style.css

```diff
@@ -0,0 +1,96 @@
+body {
+    font-family: Arial, sans-serif;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    box-sizing: border-box;
+    background-color: #f4f4f4;
+}
+
+.container {
+    max-width: 80%;
+    width: 100%;
+    padding: 20px;
+    background-color: #fff;
+    box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.1);
+    border-radius: 8px;
+}
+
+h1, h2 {
+    color: #333;
+}
+h2{
+    margin-top: 50px;
+}
+.progress-box{
+    display:  flex;
+    width: 100%;
+    height: 40px;
+    border-top: 1px solid rgb(215, 215, 215);
+    border-bottom:  1px solid rgb(215, 215, 215);
+}
+.progress-title{
+    width: 50%;
+    display: flex;
+    align-items: center;
+}
+.progress-body{
+    width: 50%;
+    display: flex;
+    align-items: center;
+}
+.progress-container {
+    width: 90%;
+    background-color: #e0e0e0;
+    border-radius: 8px;
+    height: 20px;
+}
+
+#progressBar {
+    width: 0%;
+    height: 100%;
+    background-color: #28a745;
+    border-radius: 8px;
+    transition: width 0.3s ease;
+}
+
+#progressText {
```

Comment:
why are you mixing the class names naming conventions. You are using hyphenated classes, I suggest ou do same here for consistency 
`#progress-bar`